### PR TITLE
[Traceable] New input filter and new filter values

### PR DESCRIPTION
**Pull request by mtraceable**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes:
https://github.com/demisto/content/commit/6a5634464bbc14376f11b939c99fcec1927a39a1
https://github.com/demisto/content/commit/30282a5dc37cec2edc6d86fbaaae516dbd66d6d1

## Description
- Added new input filter "IP Location Type"
- Added Another filter value "UNKNOWN" to the input "IP Reputation Level"
- Added Another filter value "UNKNOWN" to the input "IP Abuse Velocity"

## Screenshots
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by mmhw** [comment]
Hi @mtraceable,
Thank you for your contribution!
Good work :)

The code looks good!
Let's schedule a call for a Demo or if it's convenient for you send a video.
Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep).

File: Packs/Traceable/Integrations/Traceable/Traceable.py [hunk 1]
```diff
@@ -311,8 +312,35 @@ def set_threat_category_list(self, threatCategoryList):
...
+                _ipCategoriesList.append("IP_LOCATION_TYPE_HOSTING_PROVIDER")
+            elif ipCategory == "Public Proxy":
+                _ipCategoriesList.append("IP_LOCATION_TYPE_PUBLIC_PROXY")
+            elif ipCategory == "TOR Exit Node":
+                _ipCategoriesList.append("IP_LOCATION_TYPE_TOR_EXIT_NODE")
+            elif ipCategory == "BOT":
+                _ipCategoriesList.append("IP_LOCATION_TYPE_BOT")
+            else:
+                error = f"Unknown ipCategory {ipCategory} specified."
+                demisto.error(error)
```
Comment (mmhw) [inline Packs/Traceable/Integrations/Traceable/Traceable.py:332]:
```suggestion
```
It is unnecessary, since in main we already return return_error

**Review by mmhw — CHANGES_REQUESTED** [review]

File: Packs/Traceable/Integrations/Traceable/Traceable.py (hunk 1 above)
Comment (mtraceable) [reply Packs/Traceable/Integrations/Traceable/Traceable.py:332]: Done

**Review by mmhw — APPROVED** [review]
Good work!

**Pull request merged**